vault.js

```diff
@@ -1,4 +1,23 @@
 'use strict';
-module.exports = function() {
-  
+module.exports = function () {
+    var vaultKey = "secret password";
+    var vaultValue = "Nothing";
+
```

Comment:
Although this will work for one key-value pair, can you think of something that could possibly hold more than one key-value pair?